app/components/decision_component.rb

```diff
@@ -61,15 +61,15 @@ def update_path
   end
 
   def ioj_result
-    t(interests_of_justice.result, scope: [:values, :decision_result])
```

Comment:
The same values (e.g. fail) are shown as different values depending on test. (E.g. Refused v Failed)